releases-notes/v0.12.0.md

```diff
@@ -0,0 +1,6 @@
+# v0.12.0
+
+## Notable Changes
+
+1. Allow operators to override the ingress-nginx-controller name
+2. Introduce Space scoped labels that can be applied via mxe install
```

Comment:
I don't see documentation on either of these. Do need to describe how they can be used?

Reply:
updated!